### ISSUE TITLE
Folder imports

### ISSUE THREAD

**Issue by rxchard** [hardcore-sushi/DroidFS#44]
Could you add an option that makes it possible to import entire folders (recursively) without having to select every single file?

**Comment by hardcore-sushi** [comment]
Yes it should be possible but I don't have so much time right now. Your pull requests are welcome !
